docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -0,0 +1,95 @@
+# Zowe API Mediation Layer OIDC Authentication
+
+The OpenID/Connect ([OIDC](https://openid.net/specs/openid-connect-core-1_0.html)) protocol adds an authentication layer on top of the [OAuth2](https://www.rfc-editor.org/rfc/rfc6749) Authorization protocol.
+OIDC authentication, together with the z/OS [Identity Propagation](https://www.redbooks.ibm.com/redbooks/pdfs/sg247850.pdf) mechanism, is the foundation of the API ML Identity Federation.
+Zowe API ML can be configured to authenticate users by accepting Access Tokens issued by an external OIDC provider.
```

Comment:
here I think is another reference, besides IAM, IDP and OIDC, I understand this last one refers to a specific implementation though.

Reply:
Answered above.

Reply:
Added a sentence for the OIDC/OAuth2 interchangeability. IAM and IDP are not present in the text anymore.